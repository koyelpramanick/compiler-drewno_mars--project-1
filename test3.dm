and 24Kmagic bool
else false if int
give take
or perfect
return class
too hot true
today I don't feel like doing any work
void while
