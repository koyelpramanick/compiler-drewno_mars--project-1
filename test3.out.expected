AND [1,1]
MAGIC [1,5]
BOOL [1,14]
ELSE [2,1]
FALSE [2,6]
IF [2,12]
INT [2,15]
GIVE [3,1]
TAKE [3,6]
OR [4,1]
PERFECT [4,4]
RETURN [5,1]
CLASS [5,8]
FALSE [6,1]
TRUE [6,9]
EXIT [7,1]
VOID [8,1]
WHILE [8,6]
EOF [9,1]
